support-models/src/main/scala/com/gu/support/catalog/ProductOptions.scala

```diff
@@ -30,8 +30,15 @@ case object EverydayPlus extends PaperProductOptions(true)
 
 case object Everyday extends PaperProductOptions(false)
 
+sealed abstract class SupporterPlusVersionOptions extends ProductOptions
```

Comment:
Hmm, interesting, I guess conceptually you'd want anything in this ADT to 'be' a version option rather than just mixing it in so maybe slightly more correct from that point of view? But functionally no it's identical.